build/mathquill.js

```diff
@@ -2110,43 +2277,55 @@ Controller.open(function(_, super_) {
  *******************************************************/
 
 Controller.open(function(_) {
+  /**
+   * 
+   */
   _.delegateMouseEvents = function() {
     var ultimateRootjQ = this.root.jQ;
     //drag-to-select event handling
     this.container.bind('mousedown.mathquill', function(e) {
-      var ultimateRoot = Node.byId[ultimateRootjQ.attr(mqBlockId)];
-      var ultimateRootAPI = ultimateRoot.controller.API;
-
-      var rootjQ = $(e.target).closest('.mq-root-block');
+      // finds the closest parent root block to get an instance of Controller.
+      // If none exists, fallback to ultimate root controller
+      var rootjQ = $(e.target).closest('.mq-root-block'); 
       var root = Node.byId[rootjQ.attr(mqBlockId) || ultimateRootjQ.attr(mqBlockId)];
-      var ctrlr = root.controller, cursor = ctrlr.cursor, blink = cursor.blink;
-      var textareaSpan = ctrlr.textareaSpan, textarea = ctrlr.textarea;
+      var ctrlr = root.controller, 
```

Comment:
**nit** better to keep this and minimise change

Reply:
This is the build tho.